content/en/docs/v3.6/learning/api.md

```diff
@@ -360,6 +360,8 @@ Watches make three guarantees about events:
 * Reliable - a sequence of events will never drop any subsequence of events; if there are events ordered in time as a < b < c, then if the watch receives events a and c, it is guaranteed to receive b.
 * Atomic - a list of events is guaranteed to encompass complete revisions; updates in the same revision over multiple keys will not be split over several lists of events.
 
+Note that these guarantees are not the same as linarizability because the events may be received after an unspecified amount of time. These events may be already stale at the moment when they are received.
```

Comment:
I think putting term linearizability in this section might be confusing for users not familiar with it. Maybe we can just link to `content/en/docs/v3.6/learning/api_guarantees.md` if users want to read more about guarantees provided by watch.

Reply:
Changed. I thought it was already mentioned on the page more than it really is.